independent-projects/tools/codestarts/src/main/java/io/quarkus/devtools/codestarts/core/reader/QuteCodestartFileReader.java

```diff
@@ -6,30 +6,24 @@
 import java.util.Map;
 import java.util.Optional;
 import java.util.concurrent.CompletionStage;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 import org.apache.commons.io.FilenameUtils;
+import org.apache.maven.artifact.versioning.ComparableVersion;
 
 import io.quarkus.devtools.codestarts.CodestartException;
 import io.quarkus.devtools.codestarts.CodestartResource;
 import io.quarkus.devtools.codestarts.CodestartResource.Source;
-import io.quarkus.qute.CompletedStage;
-import io.quarkus.qute.Engine;
-import io.quarkus.qute.EvalContext;
-import io.quarkus.qute.Expression;
-import io.quarkus.qute.ResultMapper;
-import io.quarkus.qute.Results;
-import io.quarkus.qute.TemplateException;
-import io.quarkus.qute.TemplateLocator;
-import io.quarkus.qute.TemplateNode;
-import io.quarkus.qute.ValueResolver;
-import io.quarkus.qute.Variant;
+import io.quarkus.qute.*;
 
 final class QuteCodestartFileReader implements CodestartFileReader {
 
     private static final String TPL_QUTE_FLAG = ".tpl.qute";
     private static final String ENTRY_QUTE_FLAG = ".entry.qute";
     public static final String INCLUDE_QUTE_FLAG = ".include.qute";
     public static final String SKIP_TAG = "<SKIP>";
+    private static final Pattern VERSION_PATTERN = Pattern.compile("^(\\d+)(\\.(\\d+))?(\\.(\\d+))?.*");
```

Comment:
I guess we don't need this constant anymore and `splitVersion(...)`?

Reply:
right, duuuuuuhhhh another CI run 😬